### PR TITLE
Centaur slurm

### PR DESCRIPTION
**Pull request by THWiseman**
Small change to the test_slurm.inc.sh  script in order to get it working in GHA. Related to ticket [WX-933](https://broadworkbench.atlassian.net/browse/WX-933?atlOrigin=eyJpIjoiMWMyYTc4NWYxYWE4NDI0NGFjMWZhYWZlOTE4ZjM2ZWUiLCJwIjoiaiJ9).

[WX-933]: https://broadworkbench.atlassian.net/browse/WX-933?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/ci/bin/test_slurm.inc.sh [hunk 1]
```diff
@@ -21,21 +21,21 @@ cromwell::build::slurm::setup_slurm_environment() {
     # https://slurm.schedmd.com/
     sudo apt-get update
 
-    # Try the Lawrence Livermore National Laboratory (LLNL) version first
-    sudo apt-get install -y slurm-llnl || sudo apt-get install -y slurm-wlm
+    sudo apt-get install -y slurm-wlm
+    #As an alternative to 'slurm-wlm', you may also try 'slurm-llnl'.
+    #If you use 'slurm-llnl', change the below tee path to '/etc/slurm-llnl/slurm.conf'
+    #For reasons that are unclear, slurm-llnl doesn't work in Github Actions, and slurm-wlm doesn't work in Travis.
```
Comment (jgainerdewar) [inline src/ci/bin/test_slurm.inc.sh:27]: Thanks for adding this comment! Might be useful to add a little more context explaining that we changed this when we moved off Travis to GHA, and point to a ticket.

**Review by jgainerdewar — APPROVED** [review]

**Comment by jgainerdewar** [comment]
Haha, I see that the Slurm test failed on Travis! Can we remove it from Travis as part of this PR?

File: src/ci/bin/test_slurm.inc.sh (hunk 1 above)
Comment (THWiseman) [reply src/ci/bin/test_slurm.inc.sh]: All set, good idea!

**Comment by THWiseman** [comment]
> Haha, I see that the Slurm test failed on Travis! Can we remove it from Travis as part of this PR?

Good idea. Done.

**Review by kraefrei — APPROVED** [review]

**Pull request merged**